docs/contributing.rst

```diff
@@ -0,0 +1,28 @@
+Contributing
+############
+
+Thank you for considering to contribute to *Reading List*!
+
+.. From structlog docs:
+
+    No contribution is too small; please submit as many fixes for typos and
+    grammar bloopers as you can!
+
+:note:
+
+    Open a pull-request even if your contribution is not ready yet! We can
+    discuss and improve it together!
+
+
+Run tests
+=========
+
+::
+
+    make tests
+
+
+IRC channel
+===========
+
+Join ``#storage`` on ``irc.mozilla.org``!
```

Comment:
we're adding a general 'cloud storage' channel to talk about all things related to cloud storage